main/xcb-util-errors/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "xcb-util-errors"
+pkgver = "1.0.1"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = ["pkgconf"]
+makedepends = ["libxcb-devel"]
+pkgdesc = "XCB errors library"
+maintainer = "Isaac Freund <mail@isaacfreund.com>"
+license = "MIT"
+url = "https://xcb.freedesktop.org"
+source = f"{url}/dist/{pkgname}-{pkgver}.tar.gz"
+sha256 = "cfbd3b022bdb27a6921a4abd6b41f4071b4e4960447598abd30955d3454f4d99"
+
+
+def post_install(self):
+    self.install_license("COPYING")
+
+
+@subpackage("xcb-util-errors-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+configure_gen = []
```

Comment:
this being at the bottom of files is a relic from when gnu_configure started to reconf by default

but you also don't need it, add automake+libtool+xcb-util-macros to hostmakedepends instead

Reply:
Indeed, I should probably use my brain instead of just copying from other templates...